ui/src/index.css

```diff
@@ -46,4 +46,12 @@ header a:hover {
 #validation-grid .ag-cell {
     height: 18px;
     line-height: 18px;
+}
+
+.MuiBackdrop-root {
+    z-index: 200 !important;
+}
+
+.MuiDialog-paper {
+    z-index: 201 !important;;
```

Comment:
Better not to use the `!important` directive or specify material ui css directly. Is this something that could be done in themes instead?

Reply:
it's litteray the same rule usine z-index being override so it's above ag-grid